pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -36,11 +35,11 @@ import (
 // NodeUsage stores a node's info, pods on it, thresholds and its resource usage
 type NodeUsage struct {
 	node    *v1.Node
-	usage   map[v1.ResourceName]*resource.Quantity
+	usage   map[v1.ResourceName]float64
 	allPods []*v1.Pod
 
-	lowResourceThreshold  map[v1.ResourceName]*resource.Quantity
-	highResourceThreshold map[v1.ResourceName]*resource.Quantity
+	lowResourceThreshold  map[v1.ResourceName]float64
+	highResourceThreshold map[v1.ResourceName]float64
```

Comment:
Why can't you use `resource.Quantity`?

Reply:
Resource `capacity` multiplied by `threshold` makes a decimal number (e.g. node pods=10, parameter threshold=3 in percent, and the threshold value should be 10*3/100). 

`resource.Quantity.MilliValue()` worked well to handle the floating-point calculation issue. So changed `float64` back to `resource.Quantity`  in the follow-up commit.

Would you please help review again.